### DISCUSSION TITLE
Remove hiddenMetadataUri in contract

### DISCUSSION THREAD

**Discussion by FUCryptoClub** [hashlips-lab/nft-erc721-collection#391] [Q&A] [answered]
Hi,

Please forgive me if this is a noob question... because it actual is for me.

I don't want to do a reveal, so do I just delete the line:

` hiddenMetadataUri: 'ipfs://__CID__/hidden.json',`

in my CollectionConfig.ts

Any help or guidance will be much appreciated.

Thanks

**Comment by liarco** [comment, marked as answer]
Hi @FUCryptoClub, if you are not familiar with Solidity code, then I would suggest leaving the code as is and simply revealing the collection right after deployment.
By doing this the hidden feature is simply skipped so you can even leave the value empty or use a placeholder.

I hope this helps.

**Reply by FUCryptoClub** [reply]
Works for me. Legend
